MitigationApi.yml

```diff
@@ -598,6 +598,18 @@ components:
       - DENIED
       default: RECOMMENDED
       
+    ResolvedBy:

```

Comment:
Consider comments on other PR (fraihmwork-messages) about:

enum literals that are a bit more general and still describe what we need to do
resolvableBy instead using past tense "resolvedBy"

Reply:
From other PR:
> Maybe we need to rethink naming. These literals make sense, but not necessarily for something called 'resolvedBy'. I think we either keep the name (or 'resolvableBy' as suggested above) or change field to "resolution" and have the enum be "AUTOMATIC" or "MANUAL".